src/shared/ui/input/input.tsx

```diff
@@ -0,0 +1,8 @@
+import { Input } from "antd";
+import s from "./input.module.scss";
+
+const CustomInput = ({ ...props }) => {
```

Comment:
Стоит добавить типизацию пропсов, чтобы использовать все преимущества тайпскрипта
```ts
import { Input, InputProps } from "antd";
```

```suggestion
const CustomInput = (props: InputProps) => {
```